setup.py

```diff
@@ -0,0 +1,39 @@
+import os
+from setuptools import find_packages, setup
+
+with open(os.path.join(os.path.dirname(__file__), 'README.md')) as readme:
+    README = readme.read()
+
+# allow setup.py to be run from any path
+os.chdir(os.path.normpath(os.path.join(os.path.abspath(__file__), os.pardir)))
+
+setup(
+    name='wagtail-contact',
+    version='0.1',
+    packages=find_packages(),
+    include_package_data=True,
+    license='BSD License',  # example license
+    description='A simple Wagtail contact app.',
+    long_description=README,
+    url='https://github.com/regulusweb/wagtail-contact',
+    author='Regulus web',
+    author_email='erick@regulusweb.com',
```

Comment:
@solarissmoke  Should use `info@regulusweb.com` here?

Reply:
Yes, please use info.